fivetran/resource_webhook.go

```diff
@@ -0,0 +1,234 @@
+package fivetran
+
+import (
+	"context"
+	"fmt"
+
+	fivetran "github.com/fivetran/go-fivetran"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceWebhook() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceWebhookCreate,
+		ReadContext:   resourceWebhookRead,
+		UpdateContext: resourceWebhookUpdate,
+		DeleteContext: resourceWebhookDelete,
+		Importer:      &schema.ResourceImporter{StateContext: schema.ImportStatePassthroughContext},
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The webhook ID",
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The webhook type (group, account)",
+			},
+			"group_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The group ID",
+			},
+			"url": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Your webhooks URL endpoint for your application",
+			},
+			"events": {
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "The array of event types",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"active": {
+				Type:        schema.TypeBool,
+				Required:    true,
+				Description: "Boolean, if set to true, webhooks are immediately sent in response to events",
```

Comment:
tfplugindocs utility automatically adds field type
```suggestion
				Description: "If set to true, webhooks are immediately sent in response to events",
```